controllers/articles-controller.js

```diff
@@ -0,0 +1,15 @@
+const { selectArticle } = require('../models/articles-model');
+
+const getArticle = (request, response, next) => {
+    const {article_id} = request.params;
+
+    selectArticle(article_id).then((article) => {
+        const singleArticle = {article}
+         response.status(200).send(singleArticle); 
```

Comment:
If you want you can shorten this down so you can have `.send({ article })` and remove line 7, both ways work it's just a bit shorter